<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>connectDilog</class>
 <widget class="QDialog" name="connectDilog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Specify the initial preferences </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select the connected interfaces: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="startList"/>
     </item>
     <item>
      <widget class="QListWidget" name="endList"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
       <property name="icon">
        <iconset resource="../../netemul.qrc">
         <normaloff>:/im/images/ok.png</normaloff>:/im/images/ok.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../../netemul.qrc">
         <normaloff>:/im/images/not.png</normaloff>:/im/images/not.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../netemul.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>connectDilog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>connectDilog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>463</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
